packages/csharp/PortingAssistant/PortingAssistant.Api/Application.cs

```diff
@@ -129,6 +135,31 @@ public void SetupConnection(bool console = false)
                     return false;
                 }
             });
+
+            _connection.On<CustomerFeedbackRequest, bool>("sendCustomerFeedback", request =>
+            {
+              string uniqueMachineID = LogUploadUtils.getUniqueIdentifier();
+              request.keyname = uniqueMachineID + "/" + request.timestamp;
+              RegionEndpoint bucketRegion = RegionEndpoint.USWest2;
+              S3Upload upload = new S3Upload(
+                  bucketRegion, 
+                  "portingassistantcustomer-customerfeedbackbucketxx-1u90hz3i5ly3l",
+                  request.accessKey,
+                  request.secret
+              );
+            var contentObj = new Content
+            {
+                feedback = request.feedback,
+                category = request.category,
+                date = request.date,
+                email = request.email,
+                machineID = uniqueMachineID
+            };
+              string serializedContent = JsonConvert.SerializeObject(contentObj);  
```

Comment:
Future work log: I think we may use existing PortingAssistantJsonSerializer class for this.